core/src/components/textarea/test/standalone/textarea.e2e.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('textarea: standalone', () => {
+  /**
+   * Verifies `ion-textarea` visual display when used outside of an `ion-app` component.
+   */
+  test('should not have visual regressions', async ({ page }) => {
```

Comment:
The ASCII art hasn't rendered correctly in these:
- `core/src/components/textarea/test/standalone/textarea.e2e.ts-snapshots/textarea-standalone-diff-ios-ltr-Mobile-Firefox-linux.png`
- `core/src/components/textarea/test/standalone/textarea.e2e.ts-snapshots/textarea-standalone-diff-ios-rtl-Mobile-Firefox-linux.png`
- `core/src/components/textarea/test/standalone/textarea.e2e.ts-snapshots/textarea-standalone-diff-md-ltr-Mobile-Firefox-linux.png`
- `core/src/components/textarea/test/standalone/textarea.e2e.ts-snapshots/textarea-standalone-diff-md-rtl-Mobile-Firefox-linux.png`